bin/pubber

```diff
@@ -31,29 +31,26 @@ else
         if [[ $option == *"="* ]]; then
             k=$(echo $option | cut -d'=' -f1)
             v=$(echo $option | cut -d'=' -f2)
-            options[$k]=$v
+            if ! [[ $v =~ ^[0-9]+([.][0-9]+)?$ ]]; then
+                v="\"$v\""
+            fi
         else
-            options[$option]=$option
+            k=$option
+            v=true
         fi
+        printf -v options_json '%s"%s":%s,' "$options_json" "$k" "$v"
     done
-
-    extra_point="${options[extra_point]}"    
-    
-    if [ -n "${options[extra_field]}" ]; then
-        printf -v extra_field '"extraField": "%s",' "${options[extra_field]}"
-    fi
-    
+    options_json="{${options_json%?}}"
```

Comment:
what's the ? for?  If it's a wildcard, can you be explicit about what it's removing (trailing comma, right?)

Reply:
last character, changed to remove trailing comma